src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -19,8 +19,6 @@ import { useParams } from 'react-router-dom';
 import set from 'lodash/fp/set';
 import { Pick2 } from 'ts-multipick';
 
-import { CircleLoader } from 'src/shared/components/loader/Loader';
```

Comment:
Confused why we would remove the spinner from this component.

Could you deploy a review branch so that I can check?